### ISSUE TITLE
Keyboard aux-key navigation through app items

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#4178]
- [x] Searchbox menu navigation (PR Here #3028 and breaking changes #3827)
- [ ] Category Dropdowns in header should be navigable with arrow keys (#4636)

**Comment by developpeurweb** [comment]
@Zeyber please include the TAB key too, as they are focusable links, Category Dropdowns links in the header should be navigable with the TAB key as well.

**Comment by Zeyber** [comment]
@developpeurweb From a programatic point of view, arrow key navigation is more of subset of tab key navigation and therefore has been separated into `tab` and `arrow` key. Although, there could be a more appropriate naming convention we could use such as `tab-key` and `aux-key` to include other keys we may wish to use such as spacebar.

**Comment by Zeyber** [comment]
Closing as usefulness of this master ticket is obsoleted by documentation and the aux-key label.
